teku/src/main/java/tech/pegasys/teku/cli/subcommand/RemoteSpecLoader.java

```diff
@@ -89,6 +89,6 @@ static OkHttpValidatorRestApiClient createApiClient(final URI baseEndpoint) {
     // Strip any authentication info from the URL to ensure it doesn't get logged.
     apiEndpoint = apiEndpoint.newBuilder().username("").password("").build();
     final OkHttpClient okHttpClient = httpClientBuilder.build();
-    return new OkHttpValidatorRestApiClient(apiEndpoint, okHttpClient);
+    return new OkHttpValidatorRestApiClient(apiEndpoint, okHttpClient, false);
```

Comment:
If we can access it from here we should use `ValidatorOptions.DEFAULT_VALIDATOR_BLINDED_BLOCKS_ENABLED` here.  It won't change behaviour but will be more obviously that it doesn't care about the value when we come to remove the feature toggle.